middleware/error-handler.js

```diff
@@ -1,23 +1,17 @@
 import { StatusCodes } from 'http-status-codes';
+
 const errorHandlerMiddleware = (err, req, res, next) => {
   let customError = {
     statusCode: err.statusCode || StatusCodes.INTERNAL_SERVER_ERROR,
-    msg: err.message || 'Something went wrong, try again later'
-  };
-  if (err.name === 'ValidationError') {
-    customError.msg = Object.values(err.errors).map(val => val.message);
-    customError.statusCode = StatusCodes.BAD_REQUEST;
+    message: err.message || 'Something went wrong, try again later',
```

Comment:
Another small nice thing is making the parameters here more verbose. There is no real character limit for naming variables, so you might as well be descriptive. Good choice making it clearer.